charts/atlas-cluster/README.md

```diff
@@ -97,3 +97,27 @@ containers:
            name: my-project-atlas-cluster-admin-user
            key: connectionString.standardSrv
 ```
+
+## Notes:
+
+Atlas-operator version 0.6.1+ has to delete finalizers - this change requires additional steps.
+
+Manually workaround for the update from Atlas-cluster-0.1.7:
+1) need to remove manually the "helm.sh/hook" from Atlasproject
+
+```bash
+kubectl annotate atlasproject/opib6d6o6k helm.sh/hook-
```

Comment:
We should not use the name of the resource in this doc, this is different on each case;

We should use `--selector` to find the collection of objects to modify; we can use any labels that the `atlasproject` resource has.